src/Gui/Application.cpp

```diff
@@ -2147,12 +2148,11 @@ void Application::runApplication()
     // Activate the correct workbench
     std::string start = App::Application::Config()["StartWorkbench"];
     Base::Console().Log("Init: Activating default workbench %s\n", start.c_str());
-    std::string autoload = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/General")->
-                           GetASCII("AutoloadModule", start.c_str());
+    std::string autoload = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/General")->GetASCII("AutoloadModule", start.c_str());
```

Comment:
I'm not thrilled with the line length here, I think we should pull a reference to the group on a separate line.